src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/GetAllTest.java

```diff
@@ -35,34 +39,71 @@
 
     @Test
     public void should_return_no_data_when_empty_job_list_is_returned() throws Exception {
-        String emptyResponse = "{\"data\":[]}";
+        ResultMatcher contentIsArray = jsonPath("content").isArray();
+        ResultMatcher contentIsEmpty = jsonPath("content").isEmpty();
+        ResultMatcher pageIsLast = jsonPath("last").value(true);
+        ResultMatcher pageIsFirst = jsonPath("first").value(true);
 
-        when(jobsService.getAll(anyString())).thenReturn(new JobList(Collections.emptyList()));
+        when(jobsService.getAll(anyString(), anyInt(), anyInt()))
+            .thenReturn(new PageImpl<>(Collections.emptyList()));
 
         sendGet()
             .andExpect(status().isOk())
-            .andExpect(content().json(emptyResponse));
+            .andExpect(contentIsArray)
+            .andExpect(contentIsEmpty)
+            .andExpect(pageIsLast)
+            .andExpect(pageIsFirst);
         sendGet("/jobs?page=1")
             .andExpect(status().isOk())
-            .andExpect(content().json(emptyResponse));
+            .andExpect(contentIsArray)
+            .andExpect(contentIsEmpty)
+            .andExpect(pageIsLast)
+            .andExpect(pageIsFirst);
         sendGet("/jobs?size=1")
             .andExpect(status().isOk())
-            .andExpect(content().json(emptyResponse));
+            .andExpect(contentIsArray)
+            .andExpect(contentIsEmpty)
+            .andExpect(pageIsLast)
+            .andExpect(pageIsFirst);
         sendGet("/jobs?page=1&size=1")
             .andExpect(status().isOk())
-            .andExpect(content().json(emptyResponse));
+            .andExpect(contentIsArray)
+            .andExpect(contentIsEmpty)
+            .andExpect(pageIsLast)
+            .andExpect(pageIsFirst);
     }
 
     @Test
-    public void should_return_some_data_when_non_empty_list_is_returned() throws Exception {
+    public void should_return_default_pages_when_no_parameters_are_passed() throws Exception {
         JobData jobData = JobData.fromJob("some-id", validJob());
-        when(jobsService.getAll(anyString())).thenReturn(new JobList(Collections.singletonList(jobData)));
+        when(jobsService.getAll(anyString(), anyInt(), anyInt()))
+            .thenReturn(new PageImpl<>(Collections.singletonList(jobData)));
 
         sendGet()
             .andExpect(status().isOk())
-            .andExpect(jsonPath("data[0].action").exists())
-            .andExpect(jsonPath("data[0].id").value(jobData.id))
-            .andExpect(jsonPath("data[0].name").value(jobData.name));
+            .andExpect(jsonPath("content[0].action").exists())
+            .andExpect(jsonPath("content[0].id").value(jobData.id))
+            .andExpect(jsonPath("content[0].name").value(jobData.name))
+            .andExpect(jsonPath("totalPages").value(1))
+            .andExpect(jsonPath("totalElements").value(1))
+            .andExpect(jsonPath("numberOfElements").value(1));
```

Comment:
our api guide requires json field names to be snake_case

https://hmcts.github.io/restful-api-standards/#118

Reply:
😱

Reply:
I think there's a jackson setting for that